recipes/flatcc/all/conanfile.py

```diff
@@ -99,12 +109,36 @@ def package(self):
                       os.path.join(self.package_folder, "bin", "flatcc"))
         # Copy license file
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        # Remove cmake config files
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", "flatcc"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", "flatccruntime"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-config.cmake"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-config-version.cmake"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-targets.cmake"))
+        os.remove(os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "flatcccli-targets-release.cmake"))
+        #Patch FlatccGenerateSources.cmake file until Conan supports the flatcc:cli executable target used in FlatccGenerateSources.cmake
+        genSourcesMod = os.path.join(self.package_folder, "lib", "cmake", "flatcccli", "FlatccGenerateSources.cmake")
+        tools.replace_in_file(genSourcesMod, "flatcc::cli", "$ENV{FLATCC_CLI_EXE}", strict=True)
```

Comment:
You can do this in a patch before the build.
The script you're patching is only used in the samples and tests.
So when disabling those, the file is not used.

Reply:
Ok, I'll add a patch.
The file is also used by packages that use flatcc by the way.

Reply:
Maybe do a tools.replace (for the moment).
The branch will undergo more changes.

Reply:
I just added the patch...